### ISSUE TITLE
using sampler_softmax

### ISSUE THREAD

**Issue by MakiSzz** [tensorflow/nmt#360]
I am training a nmt from zh to en, and I have a big vocab table. To avoid oom, I need to change the batch size or using sampler softmax. What part should I change to enable sampler?
